src/components/profile/Profile.tsx

```diff
@@ -0,0 +1,5 @@
+import { Card } from "../card/Card";
+
+const Profile = Card;   // temporary shit
```

Comment:
Can you make this a placeholder component, or a functional one? Just like Card. 
Its not a lot of work, and this way we don't have the "this is a card" appearing instead of the "profile", which is confusing.